### PR TITLE
Automation testing - Skeleton

### PR DESCRIPTION
**Pull request by AntonioK180**
This is the start of the automation testing. Merging now, because I changed the machine I work on .

### CODE REVIEW

File: templates/newEvent.html [hunk 1]
```diff
@@ -62,16 +62,6 @@
...
-      {% with messages = get_flashed_messages() %}
-        {% if messages %}
-          <ul class=flashes>
-            {% for message in messages %}
-              <li>{{ message }}</li>
-            {% endfor %}
-          </ul>
-        {% endif %}
-      {% endwith %}
-
```
Comment (DaniRogachev) [inline templates/newEvent.html:74]: why removing this?

File: db_config.py [hunk 2]
```diff
@@ -10,56 +10,3 @@
...
-if users_events_check:
-    print("UsersEvents table exists.")
-else:
-    cur.execute('''CREATE TABLE UsersEvents(
-        id          INT PRIMARY KEY NOT NULL auto_increment,
-        event_id    INT NOT NULL,
-        FOREIGN KEY(event_id) REFERENCES Events(event_id),
-        user_id     INT NOT NULL,
-        FOREIGN KEY(user_id) REFERENCES Users(user_id));
-       ''')
```
Comment (DaniRogachev) [inline db_config.py:65]: why did you remove this?

**Review by DaniRogachev — COMMENTED** [review]
good job

File: db_config.py (hunk 2 above)
Comment (AntonioK180) [reply db_config.py:65]: This is not a good way of checking if tables exist. It is written in the Buld&Run page - how to create the right database with the right tables.
